lib/susedistribution.pm

```diff
@@ -59,7 +59,7 @@ sub handle_password_prompt {
     my ($console) = @_;
     $console //= '';
 
-    return if (get_var("LIVETEST") || get_var('LIVECD')) && get_var('FLAVOR') !~ /d-installer/;
+    return if (get_var("LIVETEST") || get_var('LIVECD')) && (get_var('FLAVOR') !~ /default/ && get_var('FLAVOR') !~ /ALP/);
```

Comment:
I guess you meant distri alp in the last condition?
It would be easier to use `version: agama-0.9`  with regex with `/agama/` for this condition, you need to isos post and run verification for that btw your current verification have wrong version